### PR TITLE
fix: tiktok_v2 assigning value to undefined properties

### PR DESCRIPTION
**Pull request by anantjain45823**
## What are the changes introduced in this PR?

Write a brief explainer on your code changes.

## What is the related Linear task?

Resolves INT-XXX

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

@coderabbitai review

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

File: src/v0/destinations/tiktok_ads/transformV2.js [hunk 1]
```diff
@@ -42,6 +42,8 @@ const getTrackResponsePayload = (message, destConfig, event, setDefaultForConten
   // if contents is not present but we have properties.products present which has fields with superset of contents fields
   if (!payload.properties?.contents && message.properties?.products) {
     // retreiving data from products only when contents is not present
+    // properties object may be empty due which next line may give some error
+    payload.properties = payload.properties ? payload.properties : {};
```
Comment (koladilip) [inline src/v0/destinations/tiktok_ads/transformV2.js:46]: payload.properties || {}

File: src/v0/destinations/tiktok_ads/transformV2.js [hunk 2]
```diff
@@ -55,6 +57,8 @@ const getTrackResponsePayload = (message, destConfig, event, setDefaultForConten
   }
   // setting content-type default value in case of all standard event except `page-view`
   if (!payload.properties?.content_type && setDefaultForContentType) {
+    // properties object may be empty due which next line may give some error
+    payload.properties = payload.properties ? payload.properties : {};
```
Comment (koladilip) [inline src/v0/destinations/tiktok_ads/transformV2.js:61]: payload.properties || {}

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3426/test-report.html

File: src/v0/destinations/tiktok_ads/transformV2.js (hunk 2 above)
Comment (anantjain45823) [reply src/v0/destinations/tiktok_ads/transformV2.js]: done

File: src/v0/destinations/tiktok_ads/transformV2.js (hunk 1 above)
Comment (anantjain45823) [reply src/v0/destinations/tiktok_ads/transformV2.js]: done

**Review by shrouti1507 — APPROVED** [review]

**Review by koladilip — APPROVED** [review]

**Pull request merged**